nautobot/project-static/admin/js/jquery.sortable.js

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Is this file used anywhere? I don't see it mentioned elsewhere in the diffs and it doesn't seem to be replacing an existing file in Django.